src/app/models/weather-alert.model.ts

```diff
@@ -125,6 +128,37 @@ export class WeatherAlertResponse {
       return returnedDescription;
     }
   }
+
+  /**
+   * @description - passes weather alert and loops through the event value to match pre determined key words that will be used to grab info from a local json
+   * @param {WeatherAlertInfo} weatherAlert - active passed weather alert
+   * @returns {SafetyInfo} - returns an object of weather related safety info
+   */
+  public doesWeatherAlertRelateToSafetyJobAid(weatherAlert: WeatherAlertInfo): SafetyInfo {
+    let tempTag: string;
+    const coldWeatherTags: string[] = ['Ice', 'Cold', 'Snow', 'Winter', 'Freez', 'Blizzard', 'Frost', 'Wind Chill']; // spelling of 'freez' is intentional as the event name from api may say 'freeze' or 'freezing'
+    const tropicalStormTags: string[] = ['Tropical', 'Hurricane'];
+    const highWindTags: string[] = ['High Wind', 'Tornado', 'Extreme Wind'];
+    const allTags: string[] = [...coldWeatherTags, ...tropicalStormTags, ...highWindTags]
+    if (Object.keys(weatherAlert) && weatherAlert.event) {
```

Comment:
Is it worth inverting this and doing an early exit to save an indent later? I sort of prefer the way it is but it's up to you.